firmware/init/sensor/init_map.cpp

```diff
@@ -32,13 +22,120 @@ static FunctionPointerSensor baroSensor(SensorType::BarometricPressure,
 	return baroWrapper.getBaro();
 });
 
+// This converter is shared between both fast and slow: the only difference is
+// how the *voltage* is determined, not how its converted to a pressure.
+static LinearFunc mapConverter;
+static FunctionalSensor slowMapSensor(SensorType::MapSlow, MS2NT(50));
+
+// lowest reasonable idle is maybe 600 rpm
+// one sample per cycle (1 cylinder, or "sample one cyl" mode) gives a period of 100ms
+// add some margin -> 200ms timeout for fast MAP sampling
+static FunctionalSensor fastMapSensor(SensorType::MapFast, MS2NT(200));
```

Comment:
Is it correct that the "fast" sensor has larger period than the "slow" one?
Usually frequency = 1 / period

Reply:
That 200ms period is the timeout - which in this case is how long we wait before falling back to the "slow" sensor.  The slow sensor's still updates at 200hz or whatever, but asynchronously, hence its much shorter timeout.